### DISCUSSION TITLE
I need help getting it to run, I am not code savy

### DISCUSSION THREAD

**Discussion by ajdillard99** [mathoudebine/turing-smart-screen-python#199] [Q&A]
when trying to run the configure.py I get this message ModuleNotFoundError: No module named 'psutil'. And I have really no idea what to do. If some one could walk me through step by step or has a video on setting it up that would be great.

**Comment by mathoudebine** [comment]
Hi, to run this project you need to install the required Python modules. This step should be done once, before first run.
You can follow the procedure here: https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-how-to-start The step 3 has the command to install the required modules, you will need a working Internet access

**Reply by ajdillard99** [reply]
Sorry man, I just still can't get it to work for me. Would really want it to so I can have the screen turn off when pc sleeps.

**Comment by ovni76** [comment]
Warning do not install python 3.12  i have problem with it, some module needed are not ported yet to this version, use a 3.11 python.

**Reply by ajdillard99** [reply]
when I try to run it I get no module names pustil.

**Reply by mathoudebine** [reply]
You need to open a terminal inside the parent folder "turing-smart-screen-python" (where the main.py file is located) then run the following command:
`python.exe -m pip install -r requirements.txt`
Can you post the output of this command here ? as text or screenshot
